### DISCUSSION TITLE
Feature request: add config option <ConnectKitButton showUninstalledWallets={false} />

### DISCUSSION THREAD

**Discussion by ryanberckmans** [family/connectkit#115] [Ideas]
I'm requesting a config option to attempt to hide a wallet connector in the modal if that wallet is known to be not installed.

Of course, this may only be possible for certain connectors in certain contexts. We can detect if metamask isn't installed as a browser extension, but I'm unsure if we can detect if metamask mobile is installed on a phone. And of course, it never makes sense to hide walletconnect since only the user knows if they might want to use walletconnect.

Btw, separately, it would be cool to have help code splitting connectors, so that the meat of a connector isn't loaded unless the user clicks that wallet option. Some connectors are quite heavy.

![image](https://user-images.githubusercontent.com/299324/217871142-ad6c49e4-4cfb-4063-b955-d70323a6ae89.png)

**Comment by lochie** [comment]
We currently show wallets even if not installed for the purpose of newcomers so they can be aware of what wallets exist. If we hid wallets that were not installed new people to the space would be greeted to an empty modal 😅

Regarding code splitting the connectors that is a great suggestion and is something we'll have a look at!
